src/Admin/SiteHealth.php

```diff
@@ -443,6 +444,29 @@ public function add_debug_information( $debugging_information ) {
 		);
 	}
 
+	/**
+	 * Modify test results.
+	 *
+	 * @param array $test_result Site Health test result.
+	 *
+	 * @return array Modified test result.
+	 */
+	public function modify_test_result( $test_result ) {
+		// Set the `https_status` test status to critical if `is_ssl()` returns false, along with adding to the
+		// description for why its required for AMP.
+		if (
+			isset( $test_result['test'], $test_result['status'], $test_result['description'] )
+			&& 'https_status' === $test_result['test']
+			&& 'recommended' === $test_result['status']
+			&& ! is_ssl()
```

Comment:
Is the `! is_ssl()` condition needed? Shouldn't it always be critical. 

Note that the Site Health test will _also_ show as being recommended if `is_ssl()` and yet the home URL or the site URL aren't saved with the `https:` protocol: https://github.com/WordPress/wordpress-develop/blob/8603b3da3a32634ca24023c119b77467a1cff8e1/src/wp-admin/includes/class-wp-site-health.php#L1515-L1520

```suggestion
```

Reply:
Ah, right. Addressed in https://github.com/ampproject/amp-wp/pull/4948/commits/0678063174f52e0ab7fda6ced515eba320a4549a.